Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -3710,11 +3710,11 @@ def gen2021HiMix(fragment,howMuch):
 defaultDataSets['2017Design']='CMSSW_12_0_0_pre4-113X_mc2017_design_v5-v'
 defaultDataSets['2018']='CMSSW_12_0_0_pre4-113X_upgrade2018_realistic_v5-v'
 defaultDataSets['2018Design']='CMSSW_12_0_0_pre4-113X_upgrade2018_design_v5-v'
-defaultDataSets['2021']='CMSSW_12_0_0_pre4-120X_mcRun3_2021_realistic_v2-v'
-defaultDataSets['2021Design']='CMSSW_12_0_0_pre4-120X_mcRun3_2021_design_v2-v'
-defaultDataSets['2021FS']='CMSSW_12_4_0_pre4-124X_mcRun3_2021_realistic_v1_Run3FastSim_FastSim-v'
-defaultDataSets['2023']='CMSSW_12_0_0_pre4-120X_mcRun3_2023_realistic_v2-v'
-defaultDataSets['2024']='CMSSW_12_0_0_pre4-120X_mcRun3_2024_realistic_v2-v'
+defaultDataSets['2021']='CMSSW_12_5_0_pre4-124X_mcRun3_2022_realistic_v10_BS2022-v'
+defaultDataSets['2021Design']='CMSSW_12_5_0_pre4-124X_mcRun3_2022_design_v7_design_BS2022-v'
```

Comment:
typo here? Why do we have double "design".

Reply:
right, das does not know about `dataset=/RelValTTbar_14TeV/CMSSW_12_5_0_pre4-124X_mcRun3_2022_design_v7_design_BS2022-v1/GEN-SIM` but `dataset=/RelValTTbar_14TeV/CMSSW_12_5_0_pre4-124X_mcRun3_2022_design_v7_BS2022-v1/GEN-SIM` is available though. So looks like a typo here

Reply:
Hi All, thanks for your comments. It is not a typo. It is intended to have double 'design' - we had to reproduce this wf and in order not to increment the version an additional string has been appended. The dataset exists: https://cmsweb.cern.ch/das/request?instance=prod/global&input=file+dataset%3D%2FRelValMinBias_14TeV%2FCMSSW_12_5_0_pre4-124X_mcRun3_2022_design_v7_design_BS2022-v1%2FGEN-SIM. However, dasgoclient sees no data in this dataset.

Reply:
Hi @kskovpen 
Then you need to produce full sets of it. Currently, it has only MinBias as I see on DAS, using `dataset status=* dataset=/RelVal*/CMSSW_12_5_0_pre4-124X_mcRun3_2022_design_v7_design_BS2022-v1/GEN-SIM`

The missing, for example, is TTbar.

Reply:
OK, then we need to reproduce again.